backend/main.py

```
@@ -200,104 +200,143 @@ async def apply_migrations():
-                from fix_startup_issues import JyotiFlowStartupFixer
-                startup_fixer = JyotiFlowStartupFixer()
-                await startup_fixer.fix_all_issues()
-                print("✅ Comprehensive startup fixes applied successfully")
-            except Exception as e:
-                print(f"⚠️ Comprehensive startup fixes failed: {e}")
+        # Set the pool in the db module for all routers to use
+        db.set_db_pool(db_pool)
         
-        print("🎉 JyotiFlow.ai backend startup completed successfully!")
+        print("✅ Database connection pool initialized successfully!")
+        print(f"📊 Pool configuration: connections ready for use")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)